conda.yaml

```diff
@@ -2,10 +2,10 @@ channels:
   - conda-forge
 
 dependencies:
-  - python=3.10.4
-  - imageio=2.16.1
-  - matplotlib=3.5.1
-  - scipy=1.8.0
-  - pip=20.1
+  - python=3.10.4                   # https://pyreadiness.org/3.9/ 
```

Comment:
If it works drop the python to 3.9.13 on this one as well.. 3.10 is really finicky still